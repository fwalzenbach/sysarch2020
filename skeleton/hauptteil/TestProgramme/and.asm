addiu $1, $0, 73
addiu $2, $0, 42
or $3, $1, $2