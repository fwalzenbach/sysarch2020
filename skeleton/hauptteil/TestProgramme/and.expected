@1 0x00000049
@2 0x0000002a
@3 0x00000008